### PR TITLE
Small Fix

### PR DESCRIPTION
**Pull request by sunhwapark**

### CODE REVIEW

**Comment by sunhwapark** [comment]
Some lines appear to be shifted 1 line out of place ? 

ETC_20160718_022642 
ETC_20160718_022643 
ETC_20160718_022644 
ETC_20160718_022645 
ETC_20160718_022646

File: EnglishTranslation-master/ITEM.tsv [hunk 1]
```diff
@@ -10822,7 +10822,7 @@ ITEM_20160718_010821	Holy water given by Berta, said to hold the energy of the s
 ITEM_20160723_010822	Zemyna Necklace (14 days)
 ITEM_20160723_010823	Title-Shady Geologist
 ITEM_20160723_010824	An item that allows you to obtain the "Shady Geologist"
-ITEM_20160723_010825	+2 Movement speed, +50
+ITEM_20160723_010825	Increases Movement speed by 2 and also increases physical/magic attack by 50 for 30 minutes. Right-click to use. Cooldown of 30 seconds.
```
Comment (Sunaries) [inline EnglishTranslation-master/ITEM.tsv]: Movement -> movement

File: EnglishTranslation-master/ITEM.tsv [hunk 2]
```diff
@@ -3367,7 +3367,7 @@ ITEM_20150317_003366	Dew Recepticle
 ITEM_20150317_003367	A vessel to hold water enchanted with spiritual power.
 ITEM_20150317_003368	Flimsy Charm
 ITEM_20150317_003369	Altar Base Material.
-ITEM_20150317_003370	An enemy that dies with this charm attached to it is supposed to be under our control but.. It seems kind of shabby.
+ITEM_20150317_003370	An enemy that dies with this charm attached to it is supposed to be under our control but... It seems kind of shabby.
```
Comment (Sunaries) [inline EnglishTranslation-master/ITEM.tsv]: What about

`Any enemy that dies with this charm attached to it, is supposed to be under our control... It seems kind of shabby.`
?

File: EnglishTranslation-master/QUEST_LV_0100.tsv [hunk 3]
```diff
@@ -6637,7 +6637,7 @@ QUEST_LV_0100_20150414_006636	We're looking for a stone with strange engraving l
 QUEST_LV_0100_20150414_006637	The Magic Regulator of the Royal Mausoleum can be destroyed with the power of the{nl}Activation Stones. Details can be found in the Regulatory Magic Manual.
 QUEST_LV_0100_20150414_006638	As you can see, Fedimian is right under our noses but we can't get to it with the Cockatrices {nl}around. Can you teach them a lesson on how scary humans are?
 QUEST_LV_0100_20150414_006639	Denoptic is trying to control the corrupted Guardians. {nl}Annihilate it so that the Guardians will not fall into its hands.
-QUEST_LV_0100_20150414_006640	Thank you in the name of the Goddes! {nl}Please defeat the filthy Cockatrice first that are spreading diseases.
+QUEST_LV_0100_20150414_006640	Thank you in the name of the Goddess! {nl}Please defeat the filthy Cockatrice first that are spreading diseases.
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_LV_0100.tsv:6640]: goddess

File: EnglishTranslation-master/QUEST_LV_0100.tsv [hunk 4]
```diff
@@ -15205,18 +15205,18 @@ QUEST_LV_0100_20160718_015204	So that beast has a gem stuck on it now, huh?{nl}I
 QUEST_LV_0100_20160718_015205	That red water is no one's business but our village's.{nl}Outsiders would do well to leave us alone.
 QUEST_LV_0100_20160718_015206	I'm sorry I was rude to you earlier...{nl}Please forgive this old man's insolence.
 QUEST_LV_0100_20160718_015207	May the Revelator who dreamt of chaos defeat the evil forces and bring light back into our world...{nl}If the legends are true, this red water will soon be no more.
-QUEST_LV_0100_20160718_015208	A dagger.. The owner of the dagger..{nl}I don't think that fits anyone I know..{nl}
+QUEST_LV_0100_20160718_015208	A dagger.. The owner of the dagger...{nl}I don't think that fits anyone I know..{nl}
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_LV_0100.tsv:15208]: `A dagger..` -> ellipsis
`I don't think that fits anyone I know..` -> ellipsis

File: EnglishTranslation-master/QUEST_LV_0100.tsv [hunk 5]
```diff
@@ -15205,18 +15205,18 @@ QUEST_LV_0100_20160718_015204	So that beast has a gem stuck on it now, huh?{nl}I
 QUEST_LV_0100_20160718_015205	That red water is no one's business but our village's.{nl}Outsiders would do well to leave us alone.
 QUEST_LV_0100_20160718_015206	I'm sorry I was rude to you earlier...{nl}Please forgive this old man's insolence.
 QUEST_LV_0100_20160718_015207	May the Revelator who dreamt of chaos defeat the evil forces and bring light back into our world...{nl}If the legends are true, this red water will soon be no more.
-QUEST_LV_0100_20160718_015208	A dagger.. The owner of the dagger..{nl}I don't think that fits anyone I know..{nl}
+QUEST_LV_0100_20160718_015208	A dagger.. The owner of the dagger...{nl}I don't think that fits anyone I know..{nl}
 QUEST_LV_0100_20160718_015209	I suppose we will have to go to the Masters for advice.{nl}Where did they get such a dagger and why did they do this..
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_LV_0100.tsv:15209]: ellipsis

File: EnglishTranslation-master/QUEST_LV_0100.tsv [hunk 6]
```diff
@@ -15205,18 +15205,18 @@ QUEST_LV_0100_20160718_015204	So that beast has a gem stuck on it now, huh?{nl}I
...
 QUEST_LV_0100_20160718_015206	I'm sorry I was rude to you earlier...{nl}Please forgive this old man's insolence.
 QUEST_LV_0100_20160718_015207	May the Revelator who dreamt of chaos defeat the evil forces and bring light back into our world...{nl}If the legends are true, this red water will soon be no more.
-QUEST_LV_0100_20160718_015208	A dagger.. The owner of the dagger..{nl}I don't think that fits anyone I know..{nl}
+QUEST_LV_0100_20160718_015208	A dagger.. The owner of the dagger...{nl}I don't think that fits anyone I know..{nl}
 QUEST_LV_0100_20160718_015209	I suppose we will have to go to the Masters for advice.{nl}Where did they get such a dagger and why did they do this..
-QUEST_LV_0100_20160718_015210	You've remembered something?{nl}How could you have not remembered something so important.. Well, spit it out.{nl}
+QUEST_LV_0100_20160718_015210	You've remembered something?{nl}How could you have not remembered something so important... Well, spit it out.{nl}
 QUEST_LV_0100_20160718_015211	Gytis Settlement..{nl}I think I do remember hearing something about someone doing strange things over there.{nl}
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_LV_0100.tsv:15211]: ellipsis

File: EnglishTranslation-master/QUEST_LV_0100.tsv [hunk 7]
```diff
@@ -15205,18 +15205,18 @@ QUEST_LV_0100_20160718_015204	So that beast has a gem stuck on it now, huh?{nl}I
...
 QUEST_LV_0100_20160718_015209	I suppose we will have to go to the Masters for advice.{nl}Where did they get such a dagger and why did they do this..
-QUEST_LV_0100_20160718_015210	You've remembered something?{nl}How could you have not remembered something so important.. Well, spit it out.{nl}
+QUEST_LV_0100_20160718_015210	You've remembered something?{nl}How could you have not remembered something so important... Well, spit it out.{nl}
 QUEST_LV_0100_20160718_015211	Gytis Settlement..{nl}I think I do remember hearing something about someone doing strange things over there.{nl}
 QUEST_LV_0100_20160718_015212	Here you go, I'll give you the dagger so you can find who the culprit is behind this.{nl}It'll help a great deal if you find who did this.
-QUEST_LV_0100_20160718_015213	Are you curious about the dagger again?{nl}I'm not exactly an expert in that field but.. Well, whatever. Give it here.{nl}
+QUEST_LV_0100_20160718_015213	Are you curious about the dagger again?{nl}I'm not exactly an expert in that field but... Well, whatever. Give it here.{nl}
 QUEST_LV_0100_20160718_015214	What is this..{nl}You've just come back to find me right?{nl}
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_LV_0100.tsv:15214]: ellipsis

File: EnglishTranslation-master/QUEST_LV_0100.tsv [hunk 8]
```diff
@@ -15205,18 +15205,18 @@ QUEST_LV_0100_20160718_015204	So that beast has a gem stuck on it now, huh?{nl}I
...
 QUEST_LV_0100_20160718_015215	Yes, this is Evoniphon's dagger.{nl}I don't know too much about evil energies, but it shows all of his habits.{nl}
 QUEST_LV_0100_20160718_015216	Followed a mysterious girl from Grynas Training Camp..{nl}Yes. I do seem to remember something.{nl}
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_LV_0100.tsv:15216]: ellipsis

File: EnglishTranslation-master/QUEST_LV_0100.tsv [hunk 9]
```diff
@@ -15205,18 +15205,18 @@ QUEST_LV_0100_20160718_015204	So that beast has a gem stuck on it now, huh?{nl}I
...
 QUEST_LV_0100_20160718_015217	Please let me know if anything like this happens again.{nl}I'll give you as much information as I can manage. Thank you.{nl}
 QUEST_LV_0100_20160718_015218	I've received nothing but help from you.{nl}I sincerely hope that you'll catch who ever did this...
-QUEST_LV_0100_20160718_015219	Evoniphon.. I've never heard that name before.{nl}You said something about the Ranger Master?{nl}
+QUEST_LV_0100_20160718_015219	Evoniphon... I've never heard that name before.{nl}You said something about the Ranger Master?{nl}
 QUEST_LV_0100_20160718_015220	I think I'll have to go visit sometime with something nice.{nl}Anyhow, thank you for the hard work. Farewell!
 QUEST_LV_0100_20160718_015221	I am collecting the ocean's energy here to create an altar for the goddess.{nl}But the monsters.. are causing me trouble.{nl}
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_LV_0100.tsv:15221]: ellipsis

File: EnglishTranslation-master/QUEST_LV_0100.tsv [hunk 10]
```diff
@@ -15205,18 +15205,18 @@ QUEST_LV_0100_20160718_015204	So that beast has a gem stuck on it now, huh?{nl}I
...
 QUEST_LV_0100_20160718_015222	Brother Daroul is fighting the monsters off but..{nl}They're just coming in endless waves.{nl}
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_LV_0100.tsv:15222]: ellipsis

File: EnglishTranslation-master/QUEST_LV_0100.tsv [hunk 11]
```diff
@@ -15237,7 +15237,7 @@ QUEST_LV_0100_20160718_015236	Ah well, I suppose it can't be helped.{nl}Could I
 QUEST_LV_0100_20160718_015237	There are some precious corals growing near the Kranta Skerries.{nl}Please provide the Fedimian Accessory Merchant Joana with the blue coral for the goods.{nl}
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_LV_0100.tsv:15237]: `Please provide the Fedimian Accessory Merchant Joana with the blue coral for the goods`

to

`Please provide Joana, the Accessory Merchant of Fedimian with the blue coral for the goods`

(I haven't checked the raw, but "blue coral for the goods" seems kind of meh in my opinion.)

File: EnglishTranslation-master/QUEST_LV_0100.tsv [hunk 12]
```diff
@@ -15237,7 +15237,7 @@ QUEST_LV_0100_20160718_015236	Ah well, I suppose it can't be helped.{nl}Could I
 QUEST_LV_0100_20160718_015237	There are some precious corals growing near the Kranta Skerries.{nl}Please provide the Fedimian Accessory Merchant Joana with the blue coral for the goods.{nl}
 QUEST_LV_0100_20160718_015238	I would normally craft them myself but the situation is less than optimal as you may see..{nl}I know it's a long way to go but I beg of you..
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_LV_0100.tsv:15238]: ellipsis (2x)

File: EnglishTranslation-master/QUEST_LV_0100.tsv [hunk 13]
```diff
@@ -15254,14 +15254,14 @@ QUEST_LV_0100_20160718_015253	Thank you so much... for all the help!
...
 QUEST_LV_0100_20160718_015259	Please gather the monster's energy after dealing with them.{nl}They won't bother us again if we put it near the Cranto Coast.{nl}
 QUEST_LV_0100_20160718_015260	Thank you once again for agreeing to help us.
 QUEST_LV_0100_20160718_015261	Remember, monsters from other regions don't count.{nl}Please fill it with the energy from monsters from this region.
-QUEST_LV_0100_20160718_015262	You've already filled it with energy?{nl}It would have taken me ages.. Well, I won't deny that you've got talent.{nl}
+QUEST_LV_0100_20160718_015262	You've already filled it with energy?{nl}It would have taken me ages... Well, I won't deny that you've got talent.{nl}
 QUEST_LV_0100_20160718_015263	I'll go and bury the talisman since this will only take a moment.{nl}Thank you for helping.
-QUEST_LV_0100_20160718_015264	{nl}May the blessings of Goddes Jurate be upon you..
+QUEST_LV_0100_20160718_015264	{nl}May the blessings of Goddess Jurate be upon you...
 QUEST_LV_0100_20160718_015265	Thank you for helping us get back in touch with Goddess Jurate again.{nl}This was all possible thanks to your help..{nl}
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_LV_0100.tsv:15265]: ellipsis

File: EnglishTranslation-master/QUEST_LV_0100.tsv (hunk 11 above)
Comment (sunhwapark) [inline EnglishTranslation-master/QUEST_LV_0100.tsv:15237]: I believe the blue coral might have connections to the `Greenish Stone` that was potentially named `Cerulean Stone` and  also the `Crafted Coral`. I'll make a note about it for the future and give it your suggested change.

**Comment by Sunaries** [comment]
Good to merge.

**Pull request merged**